src/main/java/thePackmaster/cards/colorlesspack/ThePrism.java

```diff
@@ -0,0 +1,79 @@
+package thePackmaster.cards.colorlesspack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.StartupCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDiscardAndDeckAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.actions.colorlesspack.PrismAction;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+import static thePackmaster.util.Wiz.att;
+
+public class ThePrism extends AbstractColorlessPackCard implements StartupCard {
+    public final static String ID = makeID("ThePrism");
+    private boolean showPreview;
+    // intellij stuff attack, all_enemy, rare, 8, 1, , , 8, 1
+
+    public ThePrism(boolean showPreview) {
+        super(ID, 3, CardType.ATTACK, CardRarity.RARE, CardTarget.ALL_ENEMY);
+        this.showPreview = showPreview;
+        baseDamage = 8;
+        baseMagicNumber = magicNumber = 8;
+        selfRetain = true;
```

Comment:
It looks like this is still marked as retaining (from an earlier draft of the card where it added to your hand, I assume). Should remove this.